package org.hartz.hartz_backend.exception.userExceptions;

public class UserNotFoundException extends RuntimeException {
}
